cli/src/complete.rs

```diff
@@ -271,6 +273,67 @@ pub fn operations() -> Vec<CompletionCandidate> {
     })
 }
 
+fn config_keys_rec(
+    prefix: ConfigNamePathBuf,
+    properties: &serde_json::Map<String, serde_json::Value>,
+    acc: &mut Vec<CompletionCandidate>,
+    only_leaves: bool,
+) {
+    for (key, value) in properties {
+        if key == "type" {
+            // used by "additionalProperties", ignore
+            continue;
+        }
```

Comment:
nit: do you remember which `"additionalProperties"` caused problems? If I understand it, only `"properties"` are scanned.

Reply:
You're right, I think I was trying to scan "additionalProperties" too at some point, but decided against it. This can be removed.